CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ and this project adheres to
 
 [#920]: https://github.com/CosmWasm/cosmwasm/issues/920
 
+### Changed
+
+- cosmwasm-std: Rename the `send` function parameter to `funds` in `WasmMsg`
+  for consistency with the wasmd message types.
```

Comment:
Could you rebase this branch on top of latest main or merge main into this branch to get the 0.14.1 secion in here (added in https://github.com/CosmWasm/cosmwasm/commit/9b3c0c4a4c2dcfe68653018cc8fa26cb925acae5)? Then we are sure this ends up in the correct block.